features/step_definitions/direct_award_steps.rb

```diff
@@ -1,9 +1,14 @@
 When(/^I have created and saved a search$/) do
   steps %Q{
-    Given I am on the /g-cloud/search page
+    Given I am on the /g-cloud/search?q=test+123 page
     And I click 'Save search'
     Then I am on the 'Choose where to save your search' page
     And I enter 'my cloud project' in the 'Name your search' field
     And I click 'Save and continue'
   }
 end
+
+
+Then(/^I should get an ODS download file$/) do
```

Comment:
This could be a parameterised general-purpose step in `common_steps.rb` - would be useful for testing CSV downloads, etc.

Reply:
This is how it is done in the "Legacy" branch: https://github.com/alphagov/digitalmarketplace-functional-tests/blob/jenkins/features/step_definitions/journey_test_steps.rb#L1329

I'm not saying that's the best way, but might be worth a look.

Reply:
But if we don't know the exact filename or expected file size it doesn't look like there's much extra there to be honest.

Reply:
I'll move this to common_steps.rb but we don't know the filename or expected size so therefore I think this is probably still the best way of doing it.